README.md

```diff
@@ -12,6 +12,7 @@ Available addons
 ----------------
 addon | version | summary
 --- | --- | ---
+[crm_automated_activity](crm_automated_activity/) | 13.0.1.0.1 | Automated Activities for CRM Stages
```

Comment:
You don't need to add it manually. This is updated by when the module is merged

Reply:
Ok